### ISSUE TITLE
plan?

### ISSUE THREAD

**Issue by rmx564354093** [koba-jon/pytorch_cpp#15]
Do you have a plan to implement Libtorch version YOLOV4 or YOLOV5 or YOLOX can be trained?

**Comment by koba-jon** [comment]
Yes, I don't know when that will be, but I plan to implement these.
My goal is to implement YOLOv3, v4, v5, X in that order.
I hope you'll be looking forward to it.

**Comment by rmx564354093** [comment]
I really need the libtorch training version of yolo5 or yoloX, but it’s a bit difficult for me to implement it myself   Looking forward to your realization

**Comment by koba-jon** [comment]
Hello,
Sorry for the wait.
Finally, I have just implemented YOLOv5!!
You can train YOLOv5 with LibTorch!

https://github.com/koba-jon/pytorch_cpp/tree/develop/v2.8.0.3/Object_Detection/YOLOv5

**Comment by koba-jon** [comment]
In addition, I implemented YOLOv8!
Please use this one!
